.gitignore

```diff
@@ -13,4 +13,7 @@ Cargo.lock
 **/.vscode/**
 
 # Ignore history files.
-**/.history/**
\ No newline at end of file
+**/.history/**
+
+# Ignore .idea
+.idea
```

Comment:
Please do not put personal `.gitignore` stuff into the global `.gitignore` of a project. Do use the private ignore list: https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file

Reply:
thanks. I've updated.